notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,418 @@
+    "image = Image.fromarray(cv2.cvtColor(img, cv2.COLOR_BGR2RGB))\n",
+    "draw_img = processing.draw_ocr_box_txt(\n",
+    "    image,\n",
+    "    boxes,\n",
+    "    txts,\n",
+    "    scores,\n",
+    "    drop_score=0.5)\n",
+    "plt.imshow(draw_img)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5eae497-e670-43f1-bbc1-2d5c850242f6",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Configuration and Pre-processing\n",
+    "Add parameter configuration and make the pre-processing."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a006146a-07c4-4b9d-9f3d-cb2f97a71aba",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Configuration\n",
+    "\n",
+    "Given the area where the reader is in the picture, and config the design size and layout information for the screen."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "af3cb3af-2ff2-4b26-96d8-71560ae6ea93",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "IMG_PATH = 'Example.jpg'\n",
+    "\n",
+    "# The coordinates of the corners of the screen\n",
+    "POINTS = [[1084.5, 64.0],    # Left top\n",
+    "          [3164.5, 160.0],   # right top\n",
+    "          [2978.0, 1791.5],  # right bottom\n",
+    "          [978.5, 1530.5]]   # left bottom\n",
+    "\n",
+    "# The size of the screen\n",
+    "DESIGN_SHAPE = (1300, 1000)\n",
+    "\n",
+    "# Output template\n",
+    "RESULT_TEMP = {\"Info_Probe\":\"探头:---\", \n",
+    "               \"Freq_Set\":\"\", \n",
+    "               \"Freq_Main\":\"null\", \n",
+    "               \"Val_Total\":\"无探头\", \n",
+    "               \"Val_X\":\"\", \n",
+    "               \"Val_Y\":\"\", \n",
+    "               \"Val_Z\":\"\", \n",
+    "               \"Unit\":\"A/m\", \n",
+    "               \"Field\":\"常规\"}\n",
+    "\n",
+    "# features and the layout information\n",
+    "DESIGN_LAYOUT = {'Info_Probe':[14, 36, 410, 135],  # feature_name, xmin, ymin, xmax, ymax\n",
+    "                 'Freq_Set':[5, 290, 544, 406], \n",
+    "                 'Val_Total':[52, 419, 1256, 741], \n",
+    "                 'Val_X':[19, 774, 433, 882], \n",
+    "                 'Val_Y':[433, 773, 874, 884], \n",
+    "                 'Val_Z':[873, 773, 1276, 883], \n",
+    "                 'Unit':[1064, 291, 1295, 403], \n",
+    "                 'Field':[5, 913, 243, 998]}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c4fc249-ba1c-40b9-889b-4bfd04cc7576",
+   "metadata": {},
+   "source": [
+    "### Pre-processing\n",
+    "\n",
+    "Use affine transformations to normalize skewed images"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "5c381bcb-04ba-49b0-b8ec-1cf90b5d4e64",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def pre_processing(img, point_list, target_shape):\n",
```

Comment:
We should add more detailed description on function definition. for example:
https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/403-action-recognition-webcam/403-action-recognition-webcam.ipynb